### DISCUSSION TITLE
v 16 tag image is not working?

### DISCUSSION THREAD

**Discussion by miconx** [tiredofit/docker-postgres#17] [Q&A]
istarted a :16 container which gives me this error message: 
 2024-03-13.12:56:13 [WARN] ** [postgres] Host '127.0.0.1' is not listening on port '5432', retrying.. (5 seconds so far)

same compose file with version 15 works ... 

image: tiredofit/postgres:15
    container_name: postgres
    volumes:
      - ./pg_logs:/logs
      - ./pg_db:/var/lib/postgresql/data
    environment:
      - TIMEZONE=Europe/Vienna
      - CONTAINER_NAME=postgres
      - SUPERUSER_PASS=superusersecret
      - DB_NAME=keycloak_db
      - DB_USER=_dbadmin
      - DB_PASS=dbpassword
      - CONTAINER_ENABLE_MONITORING=FALSE
    ports:
      - "5432:5432"

**Comment by tiredofit** [comment]
Sometimes GitHub glitches out on the build process and gives us a bad "latest" or in this case a "16" image. In this case I would recommend temporarily switching over to a more scoped tag and then trying again later on to see if the blanket tag starts working again. 16 regularly gets rebuilt on a schedule..

Current scoped tag is `:16-3.2.12`
